week1/homework/src/server.js

```diff
@@ -9,12 +9,36 @@ function createServer(port) {
   let state = 10;
 
   const server = http.createServer((request, response) => {
-    // TODO: Write your homework code here
+    console.log(request.method, request.url);
+
+    response.writeHead(200, { 'Content-Type': 'application/json' });
+    switch (request.url) {
+      case '/state':
+        response.write(JSON.stringify({ state }));
+        break;
+      case '/add':
+        state++;
```

Comment:
I appreciate that you separated app logic (addition) from technical details (stringify) 👍

Reply:
Thanks!